Patches/PlayerControlPatch.cs

```diff
@@ -725,6 +725,76 @@ class CheckVanishPatch
 {
     public static bool Prefix(PlayerControl __instance)
     {
+
+        if (AmongUsClient.Instance.IsGameOver || !AmongUsClient.Instance.AmHost)
+        {
+            return false;
+        }
+
+        // No called code if is invalid vanish
+        if (!CheckInvalidVanish(__instance))
+        {
+            return false;
+        }
+
```

Comment:
Returning false doesn't make any sense because vanilla players will constantly play the disappearing animation

Reply:
Therefore, the phantom should have a `void` function and not a `bool`

Reply:
Doesn't Check get sent by the vanilla clients to confirm vanish and appear is valid, how come making it return false wouldn't do anything?

Reply:
Try to cancel CheckVanish on vanilla client and see what happens